forms-flow-web/src/constants/constants.js

```diff
@@ -54,7 +54,11 @@ export const KEYCLOAK_REALM =
   (window._env_ && window._env_.REACT_APP_KEYCLOAK_URL_REALM) ||
   process.env.REACT_APP_KEYCLOAK_URL_REALM ||
   "forms-flow-ai";
-export const KEYCLOAK_URL =
+  export const publicWorkflowEnabled =
```

Comment:
make it all caps and refer previous boolean contstant declarations

Reply:
done